website/docs/r/mssql_server_extended_auditing_policy.html.markdown

```diff
@@ -10,7 +10,7 @@ description: |-
 
 Manages a Ms Sql Server Extended Auditing Policy.
 
-~> **NOTE:** The Server Extended Auditing Policy Can be set inline here as well as with the [mssql_server_extended_auditing_policy resource](mssql_server_extended_auditing_policy.html) resource. You can only use one or the other and using both will cause a conflict.
+~> **NOTE:** The Server Extended Auditing Policy can be set inline here as well as with the [extended_auditing_policy](extended_auditing_policy.html) block of the azurerm_mssql_database resource. You can only use one or the other and using both will cause a conflict.
```

Comment:
This note can be removed since the mssql server resource doesn't support setting an extended auditing policy
```suggestion
```